test/integration/skaffold_test.go

```diff
@@ -38,6 +38,9 @@ func TestSkaffold(t *testing.T) {
 	if NoneDriver() {
 		t.Skip("none driver doesn't support `minikube docker-env`; skaffold depends on this command")
 	}
+	if Arm64Platform() {
+		t.Skip("none driver doesn't support `minikube docker-env`; skaffold depends on this command")
```

Comment:
This doesn't make sense to me.

Reply:
copy-pasted message, my bad :( 
Removed.